### PR TITLE
feat(ui): Add open release button to issues

### PR DESCRIPTION
**Pull request by matejminar**
image

If you filter for a specific release, the button that will take you to its detail will appear.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/issueList/openInReleases.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    query
+  );
+
+  const getLastValue = (token: string[]) => {
+    if (!token) {
+      return undefined;
+    }
+
+    return token[token.length - 1];
+  };
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/issueList/openInReleases.tsx:29]: Can we pull this out into module scope?

File: src/sentry/static/sentry/app/views/issueList/openInReleases.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      }}
+      icon={<IconReleases size="sm" />}
+    >
+      {t('Open Release')}
+    </StyledButton>
+  );
+};
+
+const StyledButton = styled(Button)`
+  margin-right: ${space(0.5)};
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/issueList/openInReleases.tsx:54]: I would remove this margin and instead use a grid gap in its parent wrapper, that way it's a bit more flexible if this were ever to be moved or re-used.

**Review by billyvg — APPROVED** [review]
The button's a bit large imo, but this might be ok since you only see if there is a release in your query.

File: src/sentry/static/sentry/app/views/issueList/openInReleases.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    <StyledButton
+      to={{
+        pathname: `/organizations/${orgSlug}/releases/${encodeURIComponent(
+          releaseVersion
+        )}/`,
+        query: extractSelectionParameters(location.query),
+      }}
+      icon={<IconReleases size="sm" />}
+    >
+      {t('Open Release')}
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/issueList/openInReleases.tsx:48]: Could do `view release`

**Comment by matejminar** [comment]
We decided against this for now.

**Pull request closed without merging**